spec/activation_key_spec.rb

```diff
@@ -323,6 +323,7 @@ def hash_diff(h1, h2)
       @cp.update_activation_key(@activation_key)
     }.should raise_exception(RestClient::BadRequest)
 
+    #TODO Here should be raised exception, but validator evaluate it like it was OK
```

Comment:
My guess would be because you removed the `minLength: 1` from the ContentOverrideDTO.name field in the yaml file

Reply:
If we're going to remove the required and minLength attributes from the DTO, then we should update this test to stop expecting an exception